CHANGELOG.mdx

```diff
@@ -16,6 +16,27 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [3.3.0a1] - 2022-10-10                           
+### Deprecations and Removals
+- [#11190](https://github.com/rasahq/rasa/issues/11190): Remove code related to Rasa X local mode as it is deprecated and scheduled for removal.
+
+### Improvements
+- [#11568](https://github.com/rasahq/rasa/issues/11568): Adds new dependency pluggy, with which it was possible to implement new plugin functionality.
+  This plugin manager enables the extension and/or enhancement of the Rasa command line interface with functionality made available in the rasa-plus package.
+- [#ATO-264](https://github.com/rasahq/rasa/issues/ATO-264): Caching `Message` and `Features` fingerprints unless they are altered, saving up to 2/3 of fingerprinting time in our tests.
+- [#ATO-320](https://github.com/rasahq/rasa/issues/ATO-320): Rasa supports native installations on Apple Silicon (M1 / M2). Please
+  follow the installation instructions and take a look at the limitations.
+- [#ATO-328](https://github.com/rasahq/rasa/issues/ATO-328): Added option `--offset-timestamps-by-seconds` to offset the timestamp of events when using `rasa export`
+- [#ATO-400](https://github.com/rasahq/rasa/issues/ATO-400): Added package versions of component  dependencies as an additional part of fingerprinting calculation. Upgrading an dependency will thus lead to a retraining of the component in the future. Also, by changing fingerprint calculation, the next training after this change will be a complete retraining.
```

Comment:
These links don't work, we either have to adjust to PR number or remove the URLs.